### PR TITLE
Add back, re-configure & re-enable Scalafmt

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: internal/compiler-bridge/src/test/scala/xsbt/DependencySpecification.scala [hunk 1]
```diff
@@ -166,14 +166,8 @@ class DependencySpecification extends UnitSpec {
...
-    val classDependencies = compilerForTesting.extractDependenciesFromSrcs(srcA,
-                                                                           srcB,
-                                                                           srcC,
-                                                                           srcD,
-                                                                           srcE,
-                                                                           srcF,
-                                                                           srcG,
-                                                                           srcH)
+    val classDependencies =
+      compilerForTesting.extractDependenciesFromSrcs(srcA, srcB, srcC, srcD, srcE, srcF, srcG, srcH)
```
Comment (jvican) [inline internal/compiler-bridge/src/test/scala/xsbt/DependencySpecification.scala:170]: ❤️

**Review by jvican — DISMISSED** [review]
LGTM. Merge when this is green on the CI.

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/392/head&deploy_to_bintray=false">this link.</A>

**Comment by dwijnand** [comment]
Closing in favour of #398, which is green!

**Pull request closed without merging**